src/App.js

```diff
@@ -35,25 +50,43 @@ function App() {
       <Header />
 
       <SearchBar
+        value={searchValue}
+        waveIndex={waveIndex}
         onSearchChange={searchChangeHandler}
         onSearchSelect={searchSelectHandler}
       />
 
+      <Tabs
+        className="tabs-container"
+        selectedIndex={waveIndex}
+        onSelect={tabChangeHandler}
+      >
+        <TabList>
+          {Array.from({ length: waves }, (_, i) => (
+            <Tab key={i}>Wave {i + 1}</Tab>
+          ))}
+        </TabList>
+
+        {Array.from({ length: waves }, (_, i) => (
```

Comment:
Since the tab panels content is always the same, I was thinking if there was a way to streamline this to just 1 panel, but this method (although unsanitary) works as well.